include/swift/AST/Module.h

```diff
@@ -480,6 +480,10 @@ class ModuleDecl
     return Identifier();
   }
 
+  bool inPackage(std::string packageName) {
```

Comment:
This will create a copy of the std::string every time it's called. Please use `StringRef` or `Identifier` here.